Login/Login.css

```diff
@@ -0,0 +1,58 @@
+@import url(https://king-ui.netlify.app/styles.css);
+@import url(https://king-ui.netlify.app/Stylesheets/navbar.css);
+
+@import url(https://king-ui.netlify.app/Stylesheets/reset.css);
+
+body {
+    margin: 0;
+    display: block;
+}
+
+nav {
+    display: flex;
+    justify-content: space-around;
+    border-bottom: 2px solid var(--primary-color);
+    align-items: center;
+    background-color: white;
+}
+nav .text-input {
+    width: 50%;
+    padding: 0.5rem 1rem;
+}
+.main-wrapper {
+    display: flex;
+    width: 100%;
+    height: calc(100% - 54px);
+    justify-content: center;
+    align-items: center;
+    text-align : center;
+}
+
+.main-section {
+    display: flex;
+    flex-direction: column;
+    width: 400px;
+    height: 500px;
+    background-color: white;
+    padding: 2rem;
+    justify-content: space-evenly;
+    border-radius: 5px;
+    border: 2px solid lightgray;
+}
+.main-section .login-container .input-email {
+    padding: 5px 10px;
+    width: 70%;
+    margin-bottom: 25px;
+}
+.main-section .login-container .input-pass {
+    padding: 5px 10px;
+    width: 70%;
+    margin-bottom: 25px;
+}
+.main-section .login-container .password-helper {
+    display: flex;
+    justify-content: space-between;
+}
+.main-section .login-process .btn {
+    border: 2px solid var(--primary-color);
+}
```

Comment:
I think .main-section wont be required here...should also work without it. Like below:
.login-container .input-email

Reply:
its a good practise to follow to divide the page in sections and think in terms of components